src/lib/elements/Placeholder/Placeholder.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { Meta } from "@storybook/react";
+
+import { Placeholder } from "@/lib/elements/Placeholder/Placeholder";
+
+const meta: Meta<typeof Placeholder> = {
+  title: "elements/Placeholder",
+  component: Placeholder,
+  tags: ["autodocs"],
+  parameters: {
+    status: {
+      type: "candidate",
+    },
+  },
+};
+
+export default meta;
+
+export const TextExample = {
```

Comment:
We should start using the StoryObj annotation. It'll pick up issues such as below.

```suggestion
export const TextExample: StoryObj<typeof Placeholder> = {
```

Reply:
https://github.com/canonical/maas-react-components/pull/124